rskj-core/src/main/java/org/ethereum/core/Transaction.java

```diff
@@ -248,8 +248,6 @@ public void rlpParse() {
             byte[] r = transaction.get(7).getRLPData();
             byte[] s = transaction.get(8).getRLPData();
             this.signature = ECDSASignature.fromComponents(r, s, getRealV(v));
-        } else {
-            logger.debug("RLP encoded tx is not signed!");
```

Comment:
Although I agree this line should be removed, it was still not clear what was causing it to be logged so many times. I suggest not removing it at this stage, and I'll add a new issue to go deeper into this case.

Reply:
maybe move it to `.trace()`?

Reply:
Yes, @diega is right, we can move it to trace. If we can add a tx id it will be useful too, however this could also be part of investigating the issue.

Reply:
Ok, we keep it as trace :+1: